bin/upgrading/en.dat

```diff
@@ -2915,10 +2915,18 @@ setting.sitescheme.label=Site Skin
 
 setting.sitescheme.label=Site Scheme
 
-setting.stickyentry.error.invalid=Invalid Dreamwidth entry ID or URL entered.
+setting.stickyentry.details.label=<i>Note: blank entries will be ignored.</i>
```

Comment:
I thought we wanted to use `em` these days? And don't we try to avoid HTML in these kinds of things?